### PR TITLE
add the rest of the currency exchange rates provided by the European Central Bank

### PR DESCRIPTION
**Pull request by herlev**
adds the currencies DKK, SEK, ISK, NOK, ILS and ZAR as units to numbat

### CODE REVIEW

File: book/src/list-units.md [hunk 1]
```diff
@@ -120,6 +120,12 @@ and — where sensible — units allow for [binary prefixes](https://en.wikipedi
 | `Money` | [Thai baht](https://en.wikipedia.org/wiki/Thai_baht) | `THB`, `thai_baht`, `thai_bahts`, `฿` |
 | `Money` | [Turkish lira](https://en.wikipedia.org/wiki/Turkish_lira) | `TRY`, `turkish_lira`, `turkish_liras`, `₺` |
 | `Money` | [Japanese yen](https://en.wikipedia.org/wiki/Japanese_yen) | `JPY`, `yen`, `yens`, `¥`, `円` |
+| `Money` | [Danish krone](https://en.wikipedia.org/wiki/Danish_krone) | `DKK`, `danish_krone`, `danish_kroner` |
+| `Money` | [Swedish krona](https://en.wikipedia.org/wiki/Swedish_krona) | `SEK`, `swedish_krona`, `swedish_kronor` |
+| `Money` | [Icelandic króna](https://en.wikipedia.org/wiki/Icelandic_króna) | `ISK`, `icelandic_króna`, `icelandic_krónur` |
+| `Money` | [Norwegian krone](https://en.wikipedia.org/wiki/Norwegian_krone) | `NOK`, `norwegian_krone`, `norwegian_kroner` |
+| `Money` | [Israeli new shekel](https://en.wikipedia.org/wiki/Israeli_new_shekel) | `₪`, `ILS`, `NIS`, `israeli_new_shekel`, `israeli_new_shekels` |
+| `Money` | [South African rand](https://en.wikipedia.org/wiki/South_African_rand) | `ZAR`, `south_african_rand` |
```
Comment (hotgluebanjo) [inline book/src/list-units.md:128]: I believe we're supposed to auto-generate this table for consistent sorting: #223

File: numbat/modules/units/currencies.nbt [hunk 2]
```diff
@@ -122,3 +122,33 @@ unit singapore_dollar: Money = EUR / exchange_rate("SGD")
...
+unit danish_krone: Money = EUR / exchange_rate("DKK")
+
+@name("Swedish krona")
+@url("https://en.wikipedia.org/wiki/Swedish_krona")
+@aliases(swedish_kronor, SEK: short)
+unit swedish_krona: Money = EUR / exchange_rate("SEK")
+
+@name("Icelandic króna")
+@url("https://en.wikipedia.org/wiki/Icelandic_króna")
+@aliases(icelandic_krónur, ISK: short)
```
Comment (mhmdanas) [inline numbat/modules/units/currencies.nbt:138]: Hmm, maybe it's a good idea to add aliases without any diacritics as well?

**Review by mhmdanas — CHANGES_REQUESTED** [review]

**Review by sharkdp — APPROVED** [review]
Thank you very much for your contribution!

**Comment by sharkdp** [comment]
The changes are live on https://numbat.dev

https://numbat.dev/?q=100+DKK+to+EUR%E2%8F%8E

**Comment by sharkdp** [comment]
And released as https://github.com/sharkdp/numbat/releases/tag/v1.8.0

**Pull request merged**